nncf/quantization/algorithms/accuracy_control/backend.py

```diff
@@ -0,0 +1,166 @@
+        """
+
+    @staticmethod
+    @abstractmethod
+    def get_quantizable_metatypes() -> List[OperatorMetatype]:
+        """
+        Returns a list of metatypes for operations that may be quantized.
+
+        :return: The list of metatypes for operations that may be quantized.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def get_quantize_agnostic_metatypes() -> List[OperatorMetatype]:
+        """
+        Returns a list of quantize agnostic metatypes.
+
+        :return: The list of quantize agnostic metatypes.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def get_shape_of_metatypes() -> List[OperatorMetatype]:
+        """
+        Returns a list of shape of metatypes.
+
+        :return: The list of shape of metatypes.
+        """
+
+    # Creation of commands
+
+    @staticmethod
+    @abstractmethod
+    def create_command_to_remove_quantizer(quantizer_node: NNCFNode):
+        """
+        Creates command to remove quantizer from the quantized model.
+
+        :param quantizer_node: The quantizer which should be removed.
+        :return: The command to remove quantizer from the quantized model.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def create_command_to_update_bias(node_with_bias: NNCFNode, bias_value: Any, nncf_graph: NNCFGraph):
+        """
+        Creates command to update bias value.
+
+        :param node_with_bias: The node that corresponds to the operation with bias.
+        :param bias_value: New bias value.
+        :param nncf_graph: The NNCF graph.
+        :return: The command to update bias value.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def create_command_to_update_weight(node_with_weight: NNCFNode, weight_value: Any):
+        """
+        Creates command to update weight value.
+
+        :param node_with_weight: The node that corresponds to the operation with weight.
+        :param weight_value: New weight value.
+        :return: The command to update weight value.
+        """
+
+    # Manipulations with bias value and weights
+
+    @staticmethod
+    @abstractmethod
+    def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        """
+        Checks if the node has a bias or not.
+
+        :param node: The node to check.
+        :param nncf_graph: The NNCF graph.
+        :return: True` if `node` corresponds to the operation with bias
+            (bias is added to the output tensor of that operation), `False` otherwise.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def is_node_with_weight(node: NNCFNode) -> bool:
+        """
+        Checks if the node has a weight or not.
+
+        :param node: The node to check.
+        :param nncf_graph: The NNCF graph.
+        :return: True` if `node` corresponds to the operation with weights, `False` otherwise.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def get_bias_value(node_with_bias: NNCFNode, nncf_graph: NNCFGraph, model) -> Any:
```

Comment:
The model type hint was missed here and below.

Reply:
Done